packages/frontend/src/hooks/dashboard/useDashboard.tsx

```diff
@@ -171,17 +171,16 @@ export const useUpdateDashboardName = (
 };
 
 export const useCreateMutation = (
-    projectId: string,
+    projectUuid: string,
     showRedirectButton: boolean = false,
 ) => {
     const history = useHistory();
-    const { projectUuid } = useParams<{ projectUuid: string }>();
     const { showToastSuccess, showToastError } = useApp();
     const queryClient = useQueryClient();
     return useMutation<Dashboard, ApiError, CreateDashboard>(
-        (data) => createDashboard(projectId, data),
+        (data) => createDashboard(projectUuid, data),
         {
-            mutationKey: ['dashboard_create'],
+            mutationKey: ['dashboard_create', projectUuid],
```

Comment:
Why does this mutation key change?

Reply:
just wanted every query/mutation that depends on the project id to have it in its key